SeleniumTestXUnit/PageObjectModels/LoginPage.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using OpenQA.Selenium;
+using SeleniumTest.Core;
+using SeleniumTest.Core.Drivers;
+
+public class LoginPage : IDisposable
+{
+    public ChromeWebDriver WebDriver;
+
+    public LoginPage()
+    {
+        WebDriver = new ChromeWebDriver();
+    }
+
+    public readonly string HostUrl = "https://todo.ly";
```

Comment:
Page objects should only contain locators and logic related to that particular view, not URLs or credentials, those should only be on the config file and passed to the Page Object on the test